### PR TITLE
ignition-msgs3: update commit

### PR DESCRIPTION
**Pull request by chapulina**
This is part of https://github.com/osrf/homebrew-simulation/pull/583/files

### CODE REVIEW

File: Formula/ignition-msgs3.rb [hunk 1]
```diff
@@ -1,15 +1,15 @@
 class IgnitionMsgs3 < Formula
   desc "Middleware protobuf messages for robotics"
   homepage "https://bitbucket.org/ignitionrobotics/ign-msgs"
-  url "http://gazebosim.org/distributions/ign-msgs/releases/ignition-msgs3-3.0.0~pre4.tar.bz2"
-  version "3.0.0~pre4"
-  sha256 "f4ef46bf8fbf68f4f3aea183e3030af1d280e7783e326421b6fbe9bbeaa280a1"
+  url "https://bitbucket.org/ignitionrobotics/ign-msgs/get/2f8a9726192615852fa0eea925f96df3bac57b21.tar.gz"
+  version "3.0.0~pre4~2f8a9726"
```
Comment (scpeters) [inline Formula/ignition-msgs3.rb]: I like to add an extra integer `~1~` in between the `pre` and the checksum to ensure that new version comparisons don't depend on the value of the checksum

Comment (scpeters) [reply Formula/ignition-msgs3.rb]: like this: https://github.com/osrf/homebrew-simulation/pull/576/files#diff-91c1aff2b21ce9779185e32e591cd881R5

Comment (chapulina) [reply Formula/ignition-msgs3.rb]: Makes sense, added in https://github.com/osrf/homebrew-simulation/pull/584/commits/ff88f9877a8d830d4eda023870ea9be3291f4c03

Do I need to re-trigger the jenkins job?

Comment (scpeters) [reply Formula/ignition-msgs3.rb]: > Do I need to re-trigger the jenkins job?

yes please

Comment (chapulina) [reply Formula/ignition-msgs3.rb]: Ok, it's running:

https://build.osrfoundation.org/job/generic-release-homebrew_bottle_builder/141/

**Comment by chapulina** [comment]
Ok, it looks like the only error left is the same `brew test-bot` that's affecting #583

**Comment by scpeters** [comment]
looks good; I just updated the PR title to put the name of the formula first so it's easier to read the commit history

**Pull request merged**